spec/system/admin/rules_spec.rb

```diff
@@ -26,6 +26,8 @@
       it 'creates new record with valid attributes' do
         visit admin_rules_path
 
+        click_on I18n.t('admin.rules.add_new')
+
```

Comment:
Maybe add some assertion about something on the new page here after the navigation to that page?

`expect(page).to have_title(...)` or something which both checks we are on that page, and mitigates against spec skipping ahead before content is ready.